src/main/java/org/alfresco/mockeventgenerator/config/CamelRouteProperties.java

```diff
@@ -15,11 +15,14 @@
  */
 package org.alfresco.mockeventgenerator.config;
 
+import java.util.Map;
+
 /**
  * @author Jamal Kaabi-Mofrad
  */
 public class CamelRouteProperties
 {
+    private Map<String, String> routes;
```

Comment:
There is no need to add this property. See my comment on the _application.yaml_ file.
Basically, you need to find out where this class is being used and change it to `List<CamelRouteProperties>`